src/plugins/lua/phishing.lua

```diff
@@ -144,6 +164,12 @@ local function phishing_cb(task)
     local path = url:get_path()
     local query = url:get_query()
 
+    local excl_host = is_host_excluded(exclusion_map, host)
+    if excl_host then
+      task:insert_result(excl_symbol, 0.0, host)
```

Comment:
If you add a symbol with dynamic score `0.0` it won't be possible to set any score to it. So a rule of thumb is to add all scores with dynamic score `1.0` and set the static score to `0.0` if the final result is expected to be zero. This allows uses to configure scores as they want eventually.

Reply:
Yes, I know, but exclude list should allow only exclusion, so always should be zero? 😊, but I'm okay to put `1.0` there.

Reply:
It is always better to put 1.0 from the safety/configurability point of view.